File: 001_Alura1/SomeCoursesnCodes/Java/001/Fatorial/src/Fatorial/fatorial.java
package Fatorial;

public class fatorial {
	public static void main(String[] args) {
		int i, fatorial = 1;
		for(i = 1; i <= 10; i++) {
			fatorial *= i;
			System.out.println("O fatorial de " + i + " é: " + fatorial);
		}
	}	
}

File: _git/00_a_few_other_things/00_logics/006_fatorial/fatorial.py
"""
1) Quais dados
2) Que fazer com os dados
3) Quais as restrições
4) Qual o resultado esperado
5) Qual a sequencia de passos ao resultado
"""

numero = int(input("Digite o número para calcular o fatorial: "))

if (numero > 0):
    fatorial = 1
    for item in range(1,numero + 1):
        print("Operação {} * {}".format(fatorial,item))
        fatorial *= item

print(fatorial)
        
